src/components/studyDetails/Tables/SandboxTable.tsx

```diff
@@ -75,8 +76,20 @@ const SandboxTable: React.FC<SandboxTableProps> = ({ sandboxes, onFallBackAddres
                 data={sandboxes ?? []}
                 listItems={returnListOfItems}
                 cookiePrefix={'sandboxes-editMode' + getStudyId()}
-                disablePagination
+                disablePagination={!editMode || sandboxes.length < 10}
                 useExternalPagingAndSorting={false}
+                setExternalPerPage={undefined}
+                setExternalPageIndex={undefined}
+                setExternalSortBy={undefined}
+                externalTotalItems={undefined}
+                loading={undefined}
+                abortAndSetNewController={undefined}
+                tableToPageOne={undefined}
+                setTableToPageOne={undefined}
+                setFetchReports={undefined}
+                userHasAbortedRequest={false}
+                setUserHasAbortedRequest={undefined}
+                isFetchingAnyRequest={false}
```

Comment:
Maybe we can have these props optional and then define these with an defaultPropssnip? For instance like ParticipantTable 
![image](https://user-images.githubusercontent.com/35257765/150750438-e2dd5fac-3166-4399-a29c-9b1f5dc7c36b.png)